public/js/logout.js

```diff
@@ -1,15 +1,14 @@
 const logout = async () => {
+  const response = await fetch("/auth/logout", {
+    method: "DELETE",
+    headers: { "Content-Type": "application/json" },
+  });
 
-    const response = await fetch('/api/users/logout', {
-        method: 'POST',
-        headers: { 'Content-type': 'application/json'},
-    });
-
-    if (response.ok) {
-        document.location.replace('login');
-    } else {
-        alert(response.statusText);
-    }
+  if (response.ok) {
+    document.location.replace("/");
+  } else {
+    alert("Failed to log out.");
+  }
 };
 
-document.querySelector('#logout').addEventListener('click', logout);
\ No newline at end of file
+document.getElementById("#logout").addEventListener("click", logout);
```

Comment:
you have jQuery, why are you doing this!

Reply:
$("#logoutBtn").on("click", logout);